apps/web/src/contexts/globalLayout/GlobalSidebar/StandardSidebar.tsx

```diff
@@ -164,6 +181,12 @@ export function StandardSidebar({ queryRef }: Props) {
                 onClick={handleProfileClick}
                 icon={<UserIcon />}
               />
+              <SidebarIcon
```

Comment:
let's show Search for non-logged in users on mobile too

Reply:
Updated it!